radixdlt-java/src/main/java/com/radixdlt/client/core/address/RadixUniverseConfig.java

```diff
@@ -66,6 +67,20 @@ public RadixAddress getSystemAddress() {
 		return genesis;
 	}
 
+	public JsonObject toJson() {
+		JsonObject universe = new JsonObject();
```

Comment:
Can just do this with `RadixJson.getGson().toJsonTree(this)` no?

Reply:
Yes but if I do it that way it returns `JsonElement` instead of a `JsonObject`